src/events/cd-booking-parent-form.vue

```diff
@@ -30,11 +30,6 @@
     <p class="cd-booking-parent-form__phone-error text-danger" v-show="errors.has('phoneNumber:required')">{{ $t('Phone number is required') }}</p>
     <p class="cd-booking-parent-form__phone-error text-danger" v-show="errors.has('phoneNumber:numeric')">{{ $t('Phone number is invalid') }}</p>
 
-    <label class="cd-booking-parent-form__label" for="email">{{ $t('Email Address') }}</label>
```

Comment:
This field isn't supposed to be removed. The consensus in Pivotal, and Conor has since agreed (although I do concede that the story isn't updated) is that the field in the account section should sync with this one and be readonly. That way it's clear who the account is being created for. We could leave them separate for now and have another story to sync them up, but at the very least, this field should be still here.